index.js

```diff
@@ -0,0 +1,79 @@
+const D = require('date-fp');
+const S = require('sanctuary');
+const $ = require('sanctuary-def');
+
+const a = $.TypeVariable('a');
+const b = $.TypeVariable('b');
+
+const $TimeUnit = $.EnumType([
+  'milliseconds',
+  'seconds',
+  'minutes',
+  'hours',
+  'days']);
+
+const $TimeUnitFull = $.EnumType([
+  'milliseconds',
+  'seconds',
+  'minutes',
+  'hours',
+  'days',
+  'months',
+  'years']);
+
+const $TimeUnitGet = $.EnumType([
+  'milliseconds',
+  'seconds',
+  'minutes',
+  'hours',
+  'date',
+  'month',
+  'year']);
+
+const $Either = $.BinaryType(
+  'sanctuary/Either',
```

Comment:
I had planned to actually but I wasn't sure how to export it. Maybe as `S.EitherType`?